cvat-core/src/server-proxy.ts

```diff
@@ -742,7 +755,7 @@ class ServerProxy {
                 }
             }
             try {
-                return wait();
+                return await wait();
```

Comment:
We do not need ``await`` when return result of an async function from an async function..

Reply:
In this case we do need `await`. Without it, the function will immediately return the promise it gets from `wait`, and the `catch` block will never be entered.